src/Illuminate/Notifications/RoutesNotifications.php

```diff
@@ -46,6 +46,7 @@ public function routeNotificationFor($driver, $notification = null)
         return match ($driver) {
             'database' => $this->notifications(),
             'mail' => $this->email,
+            'default' => null,
```

Comment:
```suggestion
            default => null,
```

Match's default should not have quotes

Reply:
@axlon Fixed, thanks